memo/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+
+# Note: This crate must be built using do.sh
+
+[package]
+name = "spl-memo"
+version = "0.1.0"
+description = "Solana Program Library Memo"
+authors = ["Solana Maintainers <maintainers@solana.com>"]
+repository = "https://github.com/solana-labs/solana-program-library"
+license = "Apache-2.0"
+edition = "2018"
+
+[dependencies]
+num-derive = "0.2"
+num-traits = "0.2"
```

Comment:
Can you review these dependencies? Seems like `num-derive`, `num-traits`, and `thiserror` might no longer be needed.